app/operations/base/activity_recording.rb

```diff
@@ -1,12 +1,14 @@
 module Base
   class ActivityRecording
-    def self.create_activity(model, current_user)
-      Activity.create(
-        project: fetch_project(model),
-        user: current_user,
-        action: self.class.name.split('::').last.downcase,
-        subject: model
-      )
+    def self.create_activity(model, current_user, action)
+      Array(model).map do |record|
```

Comment:
To avoid the confusion we had before, what do you think we making it a little more clear?

```suggestion
    def self.create_activity(one_or_many_models, current_user, action)
      # this is done to ensure we accept both a single model or a collection of them
      Array(one_or_more_models).map do |model|
```

Reply:
Also, since we have three arguments now, what do you think about turning some ofthe parameters into named parameters?

```suggestion
    def self.create_activity(one_or_many_models, current_user:, action:)
```